src/App.tsx

```diff
@@ -232,6 +232,12 @@ function App(): React.ReactElement {
   }
 
   useEffect(() => {
+    const storedSessionData = sessionStorage.getItem('login')
```

Comment:
we should use a constant instead of `login`, please.  Where we define the constant, we should explain what it is being used for.